concepts/05 UI Components/TagBox/15 Create a User-Defined Item.md

```diff
@@ -1,4 +1,6 @@
-A user can select existing values and add new values to the TagBox. To enable this feature, assign **true** to the [acceptCustomValue](/api-reference/10%20UI%20Components/dxSelectBox/1%20Configuration/acceptCustomValue.md '/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#acceptCustomValue') property. Note that you need to implement the [onCustomItemCreating](/api-reference/10%20UI%20Components/dxSelectBox/1%20Configuration/onCustomItemCreating.md '/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#onCustomItemCreating') handler to create a new data source entry.
+A user can select existing values and add new values to the TagBox. To enable this feature, assign **true** to the [acceptCustomValue](/api-reference/10%20UI%20Components/dxSelectBox/1%20Configuration/acceptCustomValue.md '/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#acceptCustomValue') property. 
+
+Note that you need to implement the [onCustomItemCreating](/api-reference/10%20UI%20Components/dxSelectBox/1%20Configuration/onCustomItemCreating.md '/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#onCustomItemCreating') function to create a new data source entry. You can specify DOM events after which the component calls this function. Use the [customItemCreateEvent](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#customItemCreateEvent) property for this purpose. Besides the event passed to this property, the item can also be created when users press the **Enter** key.
```

Comment:
The last sentence here needs the same correction. If my edit to the paragraph above was correct, use it here to replace the last sentence.